processor/src/fast/sys_ops.rs

```diff
@@ -21,7 +21,7 @@ impl FastProcessor {
         &mut self,
         err_code: Felt,
         op_idx: usize,
-        host: &mut impl Host,
+        host: &mut impl AsyncHost,
```

Comment:
Here, and in other similar places, could we pass in `HostCommon` instead of `AsyncHost`.

Reply:
I will hold off on that one, since when we rebase on #1936, most of these will go away